routers/private/hook_verification.go

```diff
@@ -95,6 +96,7 @@ func readAndVerifyCommit(sha string, repo *git.Repository, env []string) error {
 				if err != nil {
 					return err
 				}
+				ctx = cache.WithCacheContext(ctx)
```

Comment:
The "cache context" should have been provided by some contexts like WebContext, why it needs to create new "cache contexts" again and again?